<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListarReservasView</class>
 <widget class="QWidget" name="ListarReservasView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F8F8F8;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 0, 206);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 800 30pt &quot;Roboto&quot;;
color: #FFFFFF;</string>
    </property>
    <property name="text">
     <string>LISTAR RESERVAS</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chkTodas">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>80</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 10pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string>Mostrar solo mis reservas efectuadas</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblReservas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>710</width>
     <height>291</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: #484848;
color: #ffffff;
font: 800 10pt &quot;Roboto&quot;;
border:none;
transition: all .5s;
}

QPushButton:hover{
	background-color: #333232
}

QPushButton:pressed{
	background-color: #484848;
}</string>
   </property>
   <property name="text">
    <string>SALIR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 12pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string>LISTA DE RESULTADOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResultados">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Roboto&quot;;</string>
   </property>
   <property name="text">
    <string>Total de resultados: 20</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
